src/cluster/config/cluster.py

```diff
@@ -50,6 +50,8 @@ def merge_service_configuration(self, overwrite_srv_cfg, default_srv_cfg):
     def validation_common(self, common_configuration):
         if "cluster-id" not in common_configuration:
             return False, "cluster-id is miss in service-configuration.yaml -> cluster -> common -> cluster-id"
+        if "cluster-type" not in common_configuration or common_configuration["cluster-type"] not in ["yarn", "k8s"]:
```

Comment:
does `cluster-type` has a default value for the previous config without this field?

Reply:
The previous config does not have cluster-type.  Please refer to forward_compatibility.py, when use old config file, use "yarn" as default cluster-type